.github/workflows/run-tutorial-ue.yml

```diff
@@ -0,0 +1,36 @@
+name: run-tutorial-ue
+on:
+  push: 

```

Comment:
would remove this

Reply:
These changes are made in https://github.com/Azure/azureml-examples/pull/199